ci/drone/functions.star

```diff
@@ -31,17 +32,17 @@ def unix_common(install_script, buildscript_to_run):
         # Install script
         download_from_boostCI(install_script, 'ci/drone'),
         # Default build script (if not exists) 
-        'if [ ! -e .drone/drone.sh ]; then %s; fi' % download_from_boostCI('drone.sh', '.drone'),
+        # 'if [ ! -e .drone/drone.sh ]; then %s; fi' % download_from_boostCI('drone.sh', '.drone'),
         # Chosen build script inside .drone (if a filename and does not exist)
-        'if [ "$(basename "{0}")" = "{0}" ] && [ ! -e .drone/{0} ]; then {1}; fi'.format(buildscript_to_run, download_from_boostCI(buildscript_to_run, '.drone')),
+        # 'if [ "$(basename "{0}")" = "{0}" ] && [ ! -e .drone/{0} ]; then {1}; fi'.format(buildscript_to_run, download_from_boostCI(buildscript_to_run, '.drone')),
```

Comment:
I would very much like to keep this. Of course what we could do is move this into the install script(s)

Reply:
move this into the install script.  yes.